<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBaseCameraConfig</class>
 <widget class="QDialog" name="DialogBaseCameraConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Base Camera Config</string>
  </property>
  <widget class="QGroupBox" name="groupBox_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>551</width>
     <height>671</height>
    </rect>
   </property>
   <property name="title">
    <string>Camera Config</string>
   </property>
   <widget class="QGroupBox" name="groupBox_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Exposure</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_exposure">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exposure_get">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>get</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_exposure_set">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>set</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_exposure">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_exposure_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_exposure_max">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_auto_exposure">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>0</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Auto </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Gain</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_gain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_gain_get">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>get</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_gain_set">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>set</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_gain">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_gain_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_gain_max">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Focus</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_focus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_focus_get">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>get</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_focus_set">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>set</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_focus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_focus_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_focus_max">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_auto_focus">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>0</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Auto</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_12">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Brightness</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_brightness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_brightness_get">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>get</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_brightness_set">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>set</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_brightness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_brightness_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_brightness_max">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_13">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Sharpness</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_sharpness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sharpness_get">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>get</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_sharpness_set">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>set</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_sharpness">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_sharpness_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_sharpness_max">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_14">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>250</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Contrast</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_contrast">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_contrast_get">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>get</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_contrast_set">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>set</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_contrast">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_contrast_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_contrast_max">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_15">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>ISO</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_iso">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_iso_get">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>get</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_iso_set">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>set</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_iso">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_iso_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_iso_max">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_16">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>470</y>
      <width>251</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Shutter Speed</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_shutter_speed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_shutter_speed_get">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>get</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_shutter_speed_set">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>31</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>set</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_shutter_speed">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_shutter_speed_min">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_shutter_speed_max">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>255</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>600</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Factory Reset</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
